src/sonic-host-services/scripts/caclmgrd

```diff
@@ -277,14 +277,18 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
 
     def is_rule_ipv4(self, rule_props):
         if (("SRC_IP" in rule_props and rule_props["SRC_IP"]) or
-           ("DST_IP" in rule_props and rule_props["DST_IP"])):
+           ("DST_IP" in rule_props and rule_props["DST_IP"]) or
+           (rule_props.get("IP_TYPE", None) == "IPV6ANY") or 
```

Comment:
I believe you should check for `"IPV4ANY"` here, not `"IPV6ANY"`

Reply:
done